portfolio/pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <properties>
     <!-- This project uses Java 8 -->
-    <maven.compiler.source>1.8</maven.compiler.source>
+	<maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
I don't know why that is showing up as indented? On my computer it looks normal and I don't think I made any changes to that part of the file so that is odd.
![Screenshot 2020-06-02 at 6 44 55 PM](https://user-images.githubusercontent.com/20546276/83586431-420e3880-a501-11ea-873c-f3bb01cfb7f4.png)

Reply:
Interesting, yeah it is marking it as a changed line so I wasn't sure if it was intentional, but it looks good in your screenshot!